api/src/main/java/marquez/api/OpenLineageResource.java

```diff
@@ -68,6 +69,11 @@ public OpenLineageResource(
   public void create(@Valid @NotNull BaseEvent event, @Suspended final AsyncResponse asyncResponse)
       throws JsonProcessingException, SQLException {
     if (event instanceof LineageEvent) {
+
+      Instant eventTime = ((LineageEvent) event).getEventTime().toInstant();
```

Comment:
Minor: Given [`LineageEvent.eventTime`](https://github.com/search?q=repo%3AMarquezProject%2Fmarquez%20getEventTime&type=code) usage, do we want to use `UTC`:

```
((LineageEvent) event).getEventTime().withZoneSameInstant(ZoneId.of("UTC")).toInstant()
```